.evergreen/find-crypt_shared.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
The crypt_shared library is named different things on different OSs, and the exact name needs to be passed into libmongocrypt.  Fortunately, the fetcher puts it as the sole file in the `lib/` directory, so this script just globs and validates that there's a single result.